custom/rch/forms.py

```diff
@@ -0,0 +1,109 @@
+from crispy_forms.helper import FormHelper
+from django import forms
+from django.utils.translation import ugettext_lazy
+
+from crispy_forms import layout as crispy
+from crispy_forms.layout import Layout, Fieldset, ButtonHolder, Submit
+
+from custom.rch.models import AreaMapping
+
+
+def get_state_names_choices():
+    states = AreaMapping.objects.order_by().values_list('stname', flat=True).distinct()
+    options = set()
+    for state in states:
+        options.add((state, state))
+    return tuple(options)
+
+
+def get_district_names_choices():
+    districts = AreaMapping.objects.order_by().values_list('dtname', flat=True).distinct()
+    options = set()
+    for district in districts:
+        options.add((district, district))
+    return tuple(options)
```

Comment:
you could do all of this in a helper function and just pass in the field you want to query